src/lobster/mcp/tools/concepts.py

```diff
@@ -0,0 +1,168 @@
+"""Concept-related tools for Lobster MCP server using FastMCP best practices."""
+
+import logging
+
+import torch
+
+from .tool_utils import _load_model
+
+logger = logging.getLogger("lobster-fastmcp-server")
+
+
+def get_sequence_concepts(model_name: str, sequences: list[str]) -> dict:
```

Comment:
nit: more specific type hint `dict[str, Any]`? If you want to be more specific, use a Pydantic model (actually I'm not completely sure whether that's supported for function output, but you could try): https://gofastmcp.com/servers/tools#pydantic-models. It would also be nice if fastmcp just supported `TypedDict`